modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,203 @@
+For topics with Tiered Storage enabled, you can mount and unmount topics to transfer the topic data between your cluster and object storage. This effectively frees up unused partition space that can be reclaimed, enables you to migrate a topic to a different cluster, and allows the topic or even the entire cluster to hibernate or be decommissioned.
+
+An unmounted topic in object storage is detached from all clusters. The original cluster releases ownership of the topic.
+
+You can mount a topic from object storage that has been previously flushed from a source cluster to a different destination cluster, or back to the same source cluster.
+
+The following data is moved to object storage when unmounting a topic from a cluster:
+
+* Topic definitions. 
+* The consumer offsets topic. This allows the destination cluster to also restore consumer group state.
+
+== Prerequisites
+
+. Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
+. xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
+
+== Unmount a topic from a cluster to object storage
+
+When you unmount a topic, all incoming writes to the topic are blocked as Redpanda migrates the topic from the cluster to object storage. Producers and consumers of the topic receive an error message indicating that the topic is no longer available. The unmounted topic is deleted in the source cluster.
+
+[tabs]
+======
+rpk::
++
+--
+In your cluster, run this command to unmount a topic to object storage:
+
+```
+rpk topic unmount <namespace>/<topic-name>
+```
+--
+Admin API::
++
+--
+To unmount a topic from a cluster using the Admin API, you must first create an outbound topic migration. The API returns the migration ID upon creation. Then, you make an API request with the migration ID to carry out the migration in steps: prepare, execute, and cut over. You must wait for each step to complete and for the <<monitor-progress,migration state>> to update before proceeding to the next step.
+
+. Make a PUT request to the `/migrations` endpoint to create a topic migration. Specify the names of the desired topics in the request body:
++
+```
+curl -X PUT http://localhost:9644/v1/migrations -d {
+  "migration_type":"outbound", 
+  "topics": [
+    {
+      "ns": "kafka", 
+      "topic": "<topic-1-name>"
+    }, 
+    {
+      "ns": "kafka", 
+      "topic": "<topic-2-name>"
+    }, 
+    {
+      "ns": "kafka", 
+      "topic": "<topic-3-name>"
+    }
+  ], "consumer_groups": []
+}
+```
++
+* `ns` is the topic namespace. This field is optional. Default value: `kafka`
+* `consumer_groups` is required.
```

Comment:
Is there information we can provide on `consumer_groups`? As far as I know, mount/unmount for consumer groups is not implemented yet. Can we explain why the field is required to create the migration?